src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/OperatorWrapCheck.java

```diff
@@ -423,12 +423,7 @@ private static DetailAST getRightNode(DetailAST node) {
             result = adjustParens(rightNode, DetailAST::getPreviousSibling);
         }
 
-        // The ARRAY_INIT AST is confusing. It should be
-        // ARRAY_INIT
-        // |--LCURLY (first child)
-        // `--RCURLY (last child)
-        // but there is no LCURLY for ARRAY_INIT
```

Comment:
@pbludov I removed your comment since condition is changed and it is not about only ARRAY_INIT anymore. Is it ok or comment should be changed?

Reply:
Yes, of course. The comment itself is also confusing. Changing `ARRAY_INIT` is very difficult and doesn't make sense.